staker-rewards/test/data/test-blockchain.json

```diff
@@ -241,6 +197,81 @@
+            "blockNumber": 1000,
+            "returnValues": {
+              "referenceID": "0x62f68c710615F2c86c2d224A1101112076FfcB7E",
+              "operator": "0xF1De9490Bf7298b5F350cE74332Ad7cf8d5cB181",
+              "sortitionPool": "0xA3748633c6786e1842b5cC44fa43db1ecC710501",
+              "amount": "20000000000000000000"
+            }
+          },
+          {
+            "name": "BondCreated",
+            "blockNumber": 1000,
+            "returnValues": {
+              "referenceID": "0x72f68c710615F2c86c2d224A1101112076FfcB7E",
+              "operator": "0xF1De9490Bf7298b5F350cE74332Ad7cf8d5cB181",
+              "sortitionPool": "0xA3748633c6786e1842b5cC44fa43db1ecC710501",
+              "amount": "5000000000000000000"
+            }
+          },
+          {
+            "name": "BondCreated",
+            "blockNumber": 1000,
+            "returnValues": {
+              "referenceID": "0x63f68c710615F2c86c2d224A1101112076FfcB7E",
+              "operator": "0x88Ed51f84c21Ae246c23137D090cdF441009D916",
+              "sortitionPool": "0xA3748633c6786e1842b5cC44fa43db1ecC710501",
+              "amount": "15000000000000000000"
+            }
+          },
+          {
+            "name": "BondCreated",
+            "blockNumber": 1000,
+            "returnValues": {
+              "referenceID": "0xEF833365aF1c921ED60491839F2dDCbc3cCE3630",
+              "operator": "0xF1De9490Bf7298b5F350cE74332Ad7cf8d5cB181",
+              "sortitionPool": "0xA3748633c6786e1842b5cC44fa43db1ecC710501",
+              "amount": "35000000000000000000"
+            }
+          },
+          {
+            "name": "BondReleased",
+            "blockNumber": 1000,
+            "returnValues": {
+              "referenceID": "0x62f68c710615F2c86c2d224A1101112076FfcB7E",
+              "operator": "0xF1De9490Bf7298b5F350cE74332Ad7cf8d5cB181"
+            }
+          },
+          {
+            "name": "BondSeized",
+            "blockNumber": 1000,
+            "returnValues": {
+              "referenceID": "0xEF833365aF1c921ED60491839F2dDCbc3cCE3630",
+              "operator": "0xF1De9490Bf7298b5F350cE74332Ad7cf8d5cB181",
+              "amount": "15000000000000000000"
+            }
+          },
+          {
+            "name": "BondReleased",
+            "blockNumber": 2000,
+            "returnValues": {
+              "referenceID": "0x72f68c710615F2c86c2d224A1101112076FfcB7E",
+              "operator": "0xF1De9490Bf7298b5F350cE74332Ad7cf8d5cB181"
+            }
+          },
+          {
+            "name": "BondCreated",
+            "blockNumber": 2000,
+            "returnValues": {
+              "referenceID": "0x64f68c710615F2c86c2d224A1101112076FfcB7E",
+              "operator": "0x88Ed51f84c21Ae246c23137D090cdF441009D916",
+              "sortitionPool": "0xA3748633c6786e1842b5cC44fa43db1ecC710501",
+              "amount": "10000000000000000000"
+            }
+          },
           {
```

Comment:
Should `UnbondedValueWithdrawn` events below have `blockNumber` set?

Reply:
Nope, that field is used just for `Bond*` events.

Reply:
Still confusing in this whole structure but OK, we do not need to fix it now, the change in this PR is completely unrelated.

Reply:
It's because we simulate events in a very simplified way in tests. Treat the events under key `"blockNumber": 2000` as "events which would be returned if I invoke `getPastEvents` with `toBlock: 2000`". Then, `blockNumber` parameter of each event determines the block in which a given event has been emitted. However, the `blockNumber` is not needed everywhere. It's needed for events that are filtered by the block number. Simulating a real mechanism will be an overkill for such simple tests.

Reply:
Yeah, that makes sense and I fully support it. What I am trying to say is that we could have block numbers on those events just for informational purposes that they happened between 1000 and 2000 block so that reading this JSON is easier when we try to match the event history so that we can evaluate numbers for assertions (this is what I tried to do).